### ISSUE TITLE
Crash Log

### ISSUE THREAD

**Issue by ardissaps** [MoonflowerTeam/pollen#55]
### What version are you seeing the problem on?

1.18.2

### Describe the issue

```markdown
https://paste.atlauncher.com/view/a2ec3f1d-9b0f-4473-a31b-9f1e86e996f5
```

### latest.log

```shell
Above
```

**Comment by jacksonhardaway** [comment]
This doesn't seem to be a Pollen issue, rather another issue occurring during Pollen event registration, specifically with static initialization of BlockModel.

I would search through your mods and see which mod causes the crash and I can reopen it if it's still an issue.

**Comment by ardissaps** [comment]
![1](https://user-images.githubusercontent.com/82106625/179141947-378f632f-576e-4d78-a112-8a4ed2c75b68.jpg)

**Comment by ardissaps** [comment]
I was installing pollen as dependency for Anchor music.

**Comment by jacksonhardaway** [comment]
One of the mods is conflicting with Pollen. You'll need to find the mod which is causing it before we can look into any fixes, but it doesn't seem to be on Pollen's side anyways.
